<launch>

	<node pkg="trajectory_mapper" name="tf_br_global" type="tf_broadcaster" output="screen">

		<param name="from_frame" value="world"/>
		<param name="to_frame" value="origin"/>

		<remap from="/T_BC" to="/T_org_vicon"/>
		<!-- <remap from="/tf_msg" to="/tf_cam0" /> -->
		<!-- <remap from="/tf_msg" to="/camera_imu/vrpn_client/raw_transform" /> -->
		<!-- <remap from="/path" to="/path_ext" /> -->
		<rosparam file="$(find trajectory_mapper)/cfg/transform_elster_origin_vicon.yaml"/> 

	</node>

	<!-- <node pkg="trajectory_mapper" name="tf_listen2" type="tf_listener" output="screen"> -->
		<!-- <param name="from_frame" value="world"/> -->
		<!-- <param name="to_frame" value="camera1"/> -->

		<!-- <remap from="/rovio/T_CW" to="/rovio/T_CW_cam1" /> -->
		<!-- <remap from="/tf_msg" to="/camera_imu/vrpn_client/raw_transform" /> -->
		<!-- <remap from="/path" to="/path_ext" /> -->

	<!-- </node> -->

</launch>
